### PR TITLE
[RE2022-264] NCBI downloader fix

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

File: src/loaders/ncbi_downloader/gtdb.py [hunk 1]
```diff
@@ -242,11 +242,12 @@ def main():
...
         print(f"\nSuccessfully downloaded {len(genome_ids)} genome files")
 
-    genome_ids_clean = set(genome_ids) - set(failed_ids)
+    # include a full set of genome_ids for each release version inside collectionssource dir
+    genome_ids_success = list(set(genome_ids_unprocessed) - set(failed_ids))
     loader_helper.create_softlinks_in_collection_source_dir(
         collection_source_dir,
         work_dir,
-        list(genome_ids_clean),
+        genome_ids_success,
```
Comment (MrCreosote) [inline src/loaders/ncbi_downloader/gtdb.py:250]: Are there any tests that need to be updated / written for this code?

**Review by MrCreosote — COMMENTED** [review]
Code changes look good to me, but @Tianhao-Gu should definitely take a look. If he approves so do I

File: src/loaders/ncbi_downloader/gtdb.py (hunk 1 above)
Comment (Xiangs18) [reply src/loaders/ncbi_downloader/gtdb.py:250]: This helper function comes from loader_helper file, which does not have any unit tests set up yet. I can write some for key functions if necessary.

Comment (MrCreosote) [reply src/loaders/ncbi_downloader/gtdb.py:250]: We're still not officially writing tests for this repo, so if there are no tests that need updating don't worry about it

Comment (Xiangs18) [reply src/loaders/ncbi_downloader/gtdb.py:250]: Then in that case, no tests need updating.

**Review by Tianhao-Gu — APPROVED** [review]
LGTM.

**Pull request merged**